extension/content.js

```diff
@@ -426,6 +442,12 @@ document.addEventListener('DOMContentLoaded', () => {
 		}).observe($('#js-pjax-container').get(0), {childList: true});
 	}
 
+	if (pageDetect.isIssue() || pageDetect.isNewIssue() || pageDetect.isPR() || pageDetect.isCompare() || pageDetect.isGist()) {
```

Comment:
Wouldn't it be better to have an explicit detection for the comment box? `pageDetect.hasCommentBox()` or something.